proxy/src/http/user.rs

```diff
@@ -305,14 +377,53 @@ impl ToDocumentedType for RegisterInput {
                 .nullable(true),
         );
 
-        document::DocumentedType::from(props).description("Input for Uesr registration")
+        document::DocumentedType::from(props).description("Input for User registration")
+    }
+}
+
+/// Bundled input data for transfer.
+#[derive(Deserialize, Serialize)]
+#[serde(rename_all = "camelCase")]
+pub struct TransferInput {
+    /// Account id of the recipient.
+    recipient: radicle_registry_client::ed25519::Public,
```

Comment:
I guess that in the future we wouldn't need to have this here since we will have the handle on the URN already and we will be able to look up the account id based on the handle?

Reply:
In the ui, either `handle` or `account_id` can be passed into the send-funds form. Depending on where/how we validate those, the input here could be also one or the other.

Reply:
Or, from the UI, if a handle is given we look up the address, that'd be another way of solving it. It would be nice to cut the duplication of having the handle in the UI and an address in the payload, which must refer to the same resource. We can think about this in a next step!